packages/dds/tree/src/feature-libraries/defaultRebaser.ts

```diff
@@ -37,6 +56,17 @@ export class DefaultRebaser implements ChangeRebaser<DefaultChangeSet, DefaultCh
     }
 }
 
+export class DefaultChangeEncoder extends ChangeEncoder<DefaultChangeSet> {
```

Comment:
this class is stateless so we can just export a single instance of it (typed ChangeEncoder<DefaultChangeSet>) instead of exporting the class and having every user construct one themselves.